USING.md

```diff
@@ -1,116 +1,210 @@
+  include the `--show-testcase` option.
+* If you want to see the messages translated into another language (see
+  "[Translation]" section below) then include e.g. `--locale da` (Docker) or
+  `--locale da_DK.UTF-8` (local installation).
+
+The same test as above with the three options included:
+
+```sh
+docker run -t --rm zonemaster/cli zonemaster.net --no-ipv6 --show-testcase --locale=da
+```
+```sh
+zonemaster-cli zonemaster.net --no-ipv6 --show-testcase --locale=da_DK.UTF-8
+```
+
+The difference between running `zonemaster-cli` on Docker or local installation
+is invocation string, `docker run -t --rm zonemaster/cli` vs. `zonemaster-cli`.
+To simplify this document, from now on the short `zonemaster-cli` will be used
+and for Docker the long string will be assumed. To simplify repeated invocation
+on Docker an alias can be created for the shell.
+
+To see all available command line options, use the `--help` command.
+
+```
+zonemaster-cli --help
+```
+
+## Test reports
+
+The severity level of the different messages is CRITICAL, ERROR, WARNING, NOTICE,
+INFO, DEBUG, DEBUG2 or DEBUG3. The default reporting level is NOTICE and higher.
+To change the level of reporting you can use a command line option, e.g
+`--level=INFO` includes level INFO and higher in the report. See
+"[Severity Level Definitions]" for more information on the levels.
+
+The default level reporting is in plain English (or some other language), but
+other output formats are also available with options `--raw` and `json`
+respectively.
+
+
+## Translation
+
+### In Docker
+
+By default all messages are in English. By using the `--locale=LANG` option
+another language can be selected. Select "LANG" from the table below to have
+Zonemaster translated into that language.
+
+LANG | Language
+-----|---------
+da   | Danish
+en   | English
+fi   | Finnish
+fr   | French
+nb   | Norwegian
+es   | Spanish
+sv   | Swedish
+
+E.g.:
+```sh
+docker run -t --rm zonemaster/cli zonemaster.net --locale=da
+```
+
+An alternative is to set the `LC_ALL` environment variable with correct language
+value when the command is invoked, which can be useful if a shell alias is
+created. E.g.
+```sh
+docker run -e LC_ALL=da -t --rm zonemaster/cli zonemaster.net
+```
+
+### In local installation
+
+By default all messages are in the language set in the local environment (if
+available in Zonemaster) or else in English. By using the `--locale=LOCALE`
+option another language can be selected. Select "LOCALE" from the table below to
+have Zonemaster translated into that language.
+
+LOCALE      | Language
+------------|---------
+da_DK.UTF-8 | Danish
+en_US.UTF-8 | English
+fi_FI.UTF-8 | Finnish
+fr_FR.UTF-8 | French
+nb_NO.UTF-8 | Norwegian
+es-CL.UTF-8 | Spanish
+sv_SE.UTF-8 | Swedish
+
+E.g.:
+```sh
+docker run -t --rm zonemaster/cli zonemaster.net --locale=da_DK.UTF-8
+```
+
+If the environment variable `LANGUAGE` is set with correct LOCALE then no option
+is needed, e.g. `LANGUAGE=da_DK.UTF-8`.
```

Comment:
Shouldn't we be saying that zonemaster-cli respects all the environment variables LANG, LC_MESSAGES, LC_ALL and LANGUAGE?